app/frontend/src/features/profile/profilePage/Home.tsx

```diff
@@ -0,0 +1,125 @@
+    justifyContent: "space-between",
+    width: "50%",
+  },
+}));
+
+interface HomeProps {
+  user: User.AsObject;
+}
+
+export default function Home({ user }: HomeProps) {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <div className={classes.info}>
+          <Typography variant="h1">{HOSTING_PREFERENCES}</Typography>
+          <LabelAndText
+            label={LAST_MINUTE}
+            text={`${user.lastMinute?.value || ""}`}
+          />
+          <LabelAndText label={MAX_GUESTS} text={`${user.maxGuests}`} />
+          <LabelAndText
+            label={ACCEPT_SMOKING}
+            text={`${user.smokingAllowed}`}
+          />
+          <LabelAndText
+            label={ACCEPT_DRINKING}
+            text={`${user.drinkingAllowed}`}
+          />
+          <LabelAndText
+            label={ACCEPT_PETS}
+            text={`${user.acceptsPets?.value || ""}`}
+          />
+          <LabelAndText
+            label={ACCEPT_KIDS}
+            text={`${user.acceptsKids?.value || ""}`}
+          />
+          <LabelAndText label={CAMPING} text={`${user.campingOk}`} />
+          <LabelAndText
+            label={WHEELCHAIR}
+            text={`${user.wheelchairAccessible?.value || ""}`}
+          />
+        </div>
+        <div className={classes.info}>
+          <Typography variant="h1">{ABOUT_HOME}</Typography>
+          <LabelAndText
+            label={SPACE}
+            text={`${user.sleepingArrangement || ""}`}
+          />
+          <LabelAndText
+            label={HAS_HOUSEMATES}
+            text={`${user.hasHousemates} ${user.housemateDetails}`}
+          />
+          <LabelAndText label={HOST_SMOKING} text={`${user.smokesAtHome}`} />
+          <LabelAndText label={HOST_DRINKING} text={`${user.drinksAtHome}`} />
+          <LabelAndText
+            label={HOST_PETS}
+            text={`${user.hasPets?.value || ""} ${user.petDetails}`}
+          />
+          <LabelAndText
+            label={HOST_KIDS}
+            text={`${user.hasKids?.value || ""} ${user.kidDetails}`}
+          />
+          <LabelAndText label={PARKING} text={`${user.parking}`} />
+          <LabelAndText label="" text="" />
+        </div>
+      </div>
+      <Divider />
+      <Typography variant="h1">{LOCAL_AREA}</Typography>
+      <Typography variant="body1">{user.area}</Typography>
+      <Divider />
+      <Typography variant="h1">{TRANSPORTATION}</Typography>
+      <Typography variant="body1">{user.parkingDetails}</Typography>
+      <Divider />
+      <Typography variant="h1">{SLEEPING_ARRANGEMENT}</Typography>
+      <Typography variant="body1">{user.sleepingDetails}</Typography>
+      <Divider />
+      <Typography variant="h1">{HOUSE_RULES}</Typography>
+      <Typography variant="body1">{user.houseRules}</Typography>
+      <Divider />
+      <Typography variant="h1">{ADDITIONAL}</Typography>
+      <Typography variant="body1">{user.otherHostInfo}</Typography>
```

Comment:
And all of these apart from `parkingDetails` 😅 . It's an enum so you might want to provide a mapping of those to more user friendly strings, otherwise I think it'll print out a meaningless numeric code.